recipes/llvmlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "llvmlite" %}
+{% set version = "0.20.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
You used `hash_type`, `hash_value` above but tried to get `sha256` here. :)

Reply:
note: This has since been fixed.